### PR TITLE
test: Add basic e2e tests for node creator

### PR DESCRIPTION
**Pull request by OlegIvaniv**
image

### CODE REVIEW

File: cypress/e2e/4-node-creator.cy.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+		nodeCreatorFeature.getters.noResults().contains('click here').click();
+		nodeCreatorFeature.getters.nodeCreatorTabs().should('exist');
+		nodeCreatorFeature.getters.getCreatorItem('Edit Image').should('exist');
+		nodeCreatorFeature.getters.selectedTab().should('have.text', 'All');
+		nodeCreatorFeature.getters.searchBar().find('button').click();
+		nodeCreatorFeature.getters.searchBar().find('input').should('be.empty')
+	})
+
+	it('should add manul trigger node', () => {
```
Comment (mutdmour) [inline cypress/e2e/4-node-creator.cy.ts:76]: typo
```suggestion
	it('should add manual trigger node', () => {
```

File: packages/editor-ui/src/components/Node/NodeCreator/SearchBar.vue [hunk 2]
```diff
@@ -32,24 +32,18 @@ export default mixins(externalHooks).extend({
...
-		eventBus: {
-			type: Object as PropType<Vue>,
-		},
 	},
 	mounted() {
-		if (this.eventBus) {
-			this.eventBus.$on("focus", this.focus);
-		}
-		setTimeout(this.focus, 0);
+		// setTimeout(this.focus, 0);
```
Comment (mutdmour) [inline packages/editor-ui/src/components/Node/NodeCreator/SearchBar.vue:37]: are the changes in this file needed?

File: packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue [hunk 3]
```diff
@@ -60,30 +61,10 @@
...
-								{{ $locale.baseText('nodeCreator.noResults.or') }}
-							</template>
-
-							<n8n-link @click="selectWebhook" v-if="[TRIGGER_NODE_FILTER, ALL_NODE_FILTER].includes(selectedType)">
-								{{ $locale.baseText('nodeCreator.noResults.webhook') }}
-							</n8n-link>
-							{{ $locale.baseText('nodeCreator.noResults.node') }}
-						</template>
-					</no-results>
+					<n8n-link @click="selectWebhook" v-if="[TRIGGER_NODE_FILTER, ALL_NODE_FILTER].includes(selectedType)">
```
Comment (mutdmour) [inline packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue:64]: why are we changing this here?

Comment (OlegIvaniv) [reply packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue:64]: Fair point, I've extracted this change to [its own PR](https://github.com/n8n-io/n8n/pull/4721).

Comment (mutdmour) [reply packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue:64]: no worries. just was not sure if intentional or not.

**Review by mutdmour — DISMISSED** [review]
LGTM.

File: cypress/pages/features/node-creator.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		activeSubcategory: () => cy.getByTestId('categorized-items-subcategory'),
+		expandedCategories: () => this.getters.creatorItem().find('>div').filter('.active').invoke('text'),
+	};
+	actions = {
+		openNodeCreator: () => {
+			this.getters.plusButton().click();
+			this.getters.nodeCreator().should('be.visible')
+		},
+			this.getters.getNthCreatorItem(n).click();
+		},
```
Comment (ivov) [inline cypress/pages/features/node-creator.ts:28]: This misresolved conflict prevents running the tests:
image

Comment (OlegIvaniv) [reply cypress/pages/features/node-creator.ts]: My bad, fixed

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.205.0`

**Pull request merged**